libsolidity/ast/Types.cpp

```diff
@@ -3090,6 +3098,25 @@ string FunctionType::canonicalName() const
 	return "function";
 }
 
+string FunctionType::humanReadableName() const
+{
+	string name;
+	switch (m_kind)
+	{
+	case Kind::Error:
+		name = "error " + m_declaration->name();
+		name += typesToString(m_parameterTypes, true);
+		break;
+	case Kind::Event:
+		name = "event " + m_declaration->name();
+		name += typesToString(m_parameterTypes, true);
+		break;
+	default:
+		name = Type::toString();
+	}
+	return name;
```

Comment:
This would be simpler without the intermediate `name` variable.
```suggestion
	switch (m_kind)
	{
	case Kind::Error:
		return "error " + m_declaration->name() + typesToString(m_parameterTypes, true);
	case Kind::Event:
		return "event " + m_declaration->name() + typesToString(m_parameterTypes, true);
	default:
		return Type::toString();
	}
```

Reply:
Done.